### DISCUSSION TITLE
Group Folder

### DISCUSSION THREAD

**Discussion by shamanix** [isontheline/pro.webssh.net#987] [Ideas]
Hello,

Thank you very much for your good work on this app.
Is it possible to add a button in the group view so I can collapse the group.
I have many groups and many more connections saved.
Sometimes I can't find the right server via the search field and then search manually in the overview. Collapsible groups would make the search much easier.

Many thanks & best regards
Robert

**Comment by isontheline** [comment]
Hello Robert @shamanix 👋

Thank you so much for reaching me out 🙏

That sounds good to me!
Could you confirm me that you want to collapse groups by tapping on their sections?
<img width="364" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/8fa394a3-b4e4-4e50-ba84-14fd86db705c">

**Comment by shamanix** [comment]
Good morning Arnaud,

Thank you very much for your answer.
Yes, that would be perfectly fine with me.
However, for users who are not waiting for this feature, a + button or something similar would be a better way to find this feature...
The programming examples already look quite good...

Best regards
Robert

**Reply by isontheline** [reply]
Thanks for your feedback @shamanix 🙏

I will implement this feature in a near release.
I will let you know when available for beta testing.

Have a nice day ☀️
